docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -143,7 +143,7 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
 
 ### Limitations
 
-- **Streaming jobs only**: The first version of Adaptive Scheduler runs with streaming jobs only. When submitting a batch job, we will automatically fall back to the default scheduler.
+- **Streaming jobs only**: The Adaptive Scheduler runs with streaming jobs only. When submitting a batch job, we will automatically fall back to the Adaptive Batch Scheduler.
```

Comment:
we will automatically fall back to the Adaptive Batch Scheduler -> Flink will use the default scheduler of batch jobs, i.e. Adaptive Batch Scheduler

Looks to me the statement of `fall back` is kind of misleading, because `Adaptive Batch Scheduler` is not the default scheduler in all cases

Reply:
Thanks a lot @zhuzhurk  for the review and good suggestions.

Sounds make sense to me, updated~